File: 백준/Bronze/1159. 농구 경기/농구 경기.cc
#include <iostream>
#include <string>
using namespace std;


int main(){
	int count;
	cin >> count;
	
	// count first alp
	int arr['z'-'a'+1] = {0,};
	for(int i =0; i < count; i++)
	{
		string s;	
		cin >> s;
		arr[s[0]-'a']++;
	}
	
	// check alp count over 5
	string ans = "";
	for(int i = 0; i < 'z'-'a'+1; i++)
	{
		if(arr[i] >= 5) ans += 'a' + i;
	}
	
	if(ans == "") ans = "PREDAJA";
	cout << ans;
	
	return 0;
}

File: 백준/Bronze/1159. 농구 경기/농구 경기.cs
using System;
using System.Collections.Generic;
class HelloWorld {
	static void Main() {
	    int n = int.Parse(Console.ReadLine());
		int[] alp = new int['z'-'a'+1];
		
		for(int i =0; i< n; i++)
		{
		    string name = Console.ReadLine();
		    alp[name[0] - 'a']++;
		}
		
		List<char> sel  = new List<char>();
		for(int i =0; i < alp.Length; i++)
		{
		    if(alp[i]>=5) sel.Add((char)('a'+i));
		}
		
		if(sel.Count==0)
		{
		    Console.WriteLine("PREDAJA");
		    return;
		}
		
		sel.Sort();
		Console.WriteLine(string.Join("", sel));
	}
}